internal/mesh/internal/types/upstreams.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 const (
-	UpstreamsKind = "Upstreams"
```

Comment:
Did you want to save this for a following PR, or expand the scope of this one to fully rename it?

Reply:
Yep! I'll do the full rename in a later PR.

Reply:
Actually this got caught up in my IDE renaming too many things. I'll fix it.